h2o-r/h2o-package/R/connection.R

```diff
@@ -841,7 +838,8 @@ h2o.resume <- function(recovery_dir=NULL) {
   cat("Performing one-time download of h2o.jar from\n")
   cat("    ", h2o_url, "\n")
   cat("(This could take a few minutes, please be patient...)\n")
-  download.file(url = h2o_url, destfile = temp_file, mode = "wb", cacheOK = FALSE, quiet = TRUE)
+  flush.console()
+  download.file(url = h2o_url, destfile = temp_file, mode = "wb", cacheOK = FALSE, quiet = TRUE, timeout = max(1000, getOption("timeout")))
```

Comment:
timeout is in seconds, do we really want the user to wait up to 1000 seconds for everything to fail?

Reply:
@michalkurka That was the exact timeout duration that CRAN suggested, based on the size of the h2o.jar, so I think that's probably what we should leave it as.